docs/source/model_design/programming_vivarium/archiving_models/index.rst

```diff
@@ -31,7 +31,81 @@ Archiving Models
    :local:
    :depth: 1
 
+The archival process is documented quite well by engineering on this `archiving a simulation Hub page <https://hub.ihme.washington.edu/pages/viewpage.action?spaceKey=SSE&title=Archiving+a+Simulation+Model+Repository>`_. Therefore, 
+this page will not go into as much depth to avoid duplicating information.
+
+Archiving a project is the process of storing information and data 
+in order to make it publically available. Zenodo is an open repository 
+we use to accomplish this. Basically, archiving a repository is 
+adding data to the repo, checking to make sure the informaiton is 
+intelligible and correct for non-simulation science individuals, and 
+creating a referenceable version of the GitHub repo on Zenodo. 
+
+The archiving process can be summarized in 4 steps: 
+
+#. Adding artifacts 
+#. Updating repository information
+#. Running a replication test 
+#. Obtaining a DOI from Zenodo 
+
+Some more background and callouts for each step is included below. 
+
+Adding Artifacts
+----------------
+
+When building simulations, we store artifacts on the cluster not on 
+the GitHub repository because storing large files on GitHub is difficult 
+and can make cloning and working on the repo much slower. 
+Therefore, once the model is complete we need to add the artifacts 
+to repo so that external users can run the model without cluster access. 
+
+This process is well documented on the `archiving a simulation Hub page <https://hub.ihme.washington.edu/pages/viewpage.action?spaceKey=SSE&title=Archiving+a+Simulation+Model+Repository>`_. 
+With some helpful tips on the `sim sci tip sheet <https://hub.ihme.washington.edu/pages/viewpage.action?pageId=234493419>`_. 
+
+Updating Repository Information
+-------------------------------
+
+There are 2 files you should check for accuracy at this point: 
+
+#. :code:`.zenodo.json` which needs to include the Orcid IDs for everyone who worked on the project 
+#. :code:`README.rst` which needs to be accurate to the actual steps needed for running a simulation. The goal is that someone can read these instructions and successfully clone the repo and run a simulation. This file is best updated in tandem with step 3 - running a replication test. 
+
 .. todo::
 
-  Add to this page. Include how-to type information on archiving models, targeted to a research team audience. Include links to the engineering documentation as well and the `Hub page maintained by engineering <https://hub.ihme.washington.edu/pages/viewpage.action?spaceKey=SSE&title=Archiving+a+Simulation+Model+Repository>`_. 
+  Add link to Orcid ID reference that I am fairly confident was created based on `this slack thread <https://ihme.slack.com/archives/C221EE33J/p1704324609813369>`_ but can't locate. 
```

Comment:
I believe it is stored on the cluster at `/mnt/team/simulation_science/priv/orcid.json`.

Reply:
Wow... the number of times I've asked for this...